k2/torch/csrc/deserialization.h

```diff
@@ -22,9 +22,27 @@
 #include <string>
 
 #include "k2/csrc/fsa.h"
+#include "k2/torch/csrc/utils.h"
 
 namespace k2 {
 
+void RegisterRaggedInt();
+
+// A helper class to construct a Ragged<int32_t> from an archive
+// TODO(fangjun): Make it a template
+struct RaggedIntHelper : public Ragged<int32_t>,
+                         public torch::CustomClassHolder {
+  using k2::Ragged<int32_t>::Ragged;
+  RaggedIntHelper(Ragged<int32_t> ragged) : Ragged<int32_t>(ragged) {}
+};
+
+struct RaggedRegister {
+  RaggedRegister() {
+    static std::once_flag register_ragged_int_flag;
```

Comment:
The `once_flag` and `call_once` are not needed if this class is moved to `.cu` file.

If it is invoked multiple times, it will throw in the second invocation.